Assets/MRTK/Core/Utilities/Lines/Renderers/MixedRealityLineRenderer.cs

```diff
@@ -83,6 +83,14 @@ public float FadeLineAnimationTime
             get => fadeLineAnimationTime;
             set => fadeLineAnimationTime = value;
         }
+        
+        /// <summary>
+        /// Gets the LineRenderer points
+        /// </summary>
+        public Vector3[] Positions
+        {
+            get => positions;
+        }
```

Comment:
Do you need to edit the underlying array? Or just need access to the data?

If you just need access, I recommend this property returns an `IReadOnlyList` instead, to align with [CA1819](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1819) and inform the caller that the list shouldn't be changed.

```suggestion
        public IReadOnlyList<Vector3> Positions => positions;
```

(if you accept this change directly, you'll need to add `using System.Collections.Generic;` as well manually)

Reply:
yes, I just need underlying data

Reply:
@keveleigh, I have some other commits after 1776dc4. I just wanted them to exclude from this pull request. Can you help me on this? Thanks

Reply:
@SGHolospace Yep! It looks like, from your branch, you should be able to call `git reset 1776dc4140dd694a23a66276e28e8a889075b807`, which will reset your branch back to that commit https://github.com/microsoft/MixedRealityToolkit-Unity/pull/8850/commits/1776dc4140dd694a23a66276e28e8a889075b807. Then, you'll need to call `git push -f` to force push your remote branch and remove the two extra commits.